src/our-work.html

```diff
@@ -95,7 +95,7 @@ <h2 class="mb-4">Featured work</h2>
         <div class="col">
           <div class="card h-100 text-dark">
             <div class="card-body px-lg-5 mx-lg-3 py-lg-5 my-lg-5 pt-3 mt-3 pb-5 pb-3 px-4 px-2">
-              <div class="not-outcome" style="min-height: 70%">
+              <div class="min-h-70">
```

Comment:
I based this naming off of this class https://getbootstrap.com/docs/5.3/utilities/sizing/#relative-to-the-viewport

Reply:
The docs there say:

> Width and height utilities are generated from the utility API in _utilities.scss. Includes support for 25%, 50%, 75%, 100%, and auto by default. Modify those values as you need to generate different utilities here.

Do we need CSS for the 70% height here?

Reply:
Yea there aren't any min-height percentage classes, only min-height viewport ones. The quote from the Doc refers to height and width classes, not min-height ones.

Reply:
Hmm ok... but it still looks like we are missing a style rule then? I don't see any CSS for `.min-h-70` and I don't think that class is taking affect:

![image](https://github.com/compilerla/compiler.la/assets/1783439/5665a8fe-4fe4-4ed7-abe2-a9d291fdc91f)

Reply:
Oops - just fixed this with https://github.com/compilerla/compiler.la/pull/220/commits/57d202cafe0c365ae95fb3496e753986103f6b6a

Can you retest?

Reply:
Looks good :+1: